infra/media/thumbnailMaker/KImageMagickCropper.php

```diff
@@ -20,6 +20,10 @@ class KImageMagickCropper extends KBaseCropper
 	 * 
```

Comment:
Why did you change the file permission?

Reply:
Added

Reply:
I changed the permission for the file on my machine so I'll be able to save changes, otherwise I wasn't able to edit the code.

Reply:
Even if you did that to work, dont commit the file permission change.

Reply:
I've tried to restore the permissions to the original. Is it right now?